backend/test/dataset/test_create_dataset.py

```diff
@@ -88,7 +89,7 @@ def test_create_dataset_success(mock_s3, mock_dataset_dao, dataset_type: str, sc
 
     dataset_location = f's3://{TEST_ENV_CONFIG["DATA_BUCKET"]}/{directory_name}'
 
-    test_dataset = generate_dataset_model(event=mock_event, scope=scope, dataset_location=dataset_location)
+    test_dataset = generate_dataset_model(event=mock_event, scope=scope, dataset_location=dataset_location, type=dataset_type)
```

Comment:
Not that we expect it to change anything but for good measure this is a parameterized test so we should add GROUP to it probably right?

Reply:
Yeah good idea